Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -11424,6 +11424,26 @@ - (void)dropInteraction:(UIDropInteraction *)interaction sessionDidEnd:(id<UIDro
 
 #endif // ENABLE(DRAG_SUPPORT)
 
+#if HAVE(UITOOLTIPINTERACTION)
+- (void)_toolTipChanged:(NSString *)newToolTip {
+    if (!_toolTip) {
+        _toolTip = adoptNS([[UIToolTipInteraction alloc] init]);
+        [_toolTip setDelegate:self];
+        [self addInteraction:_toolTip.get()];
+    }
+
+    [_toolTip setDefaultToolTip:newToolTip];
+    RetainPtr sceneView = [[UINSSharedApplicationDelegate() hostWindowForUIWindow:[self window]] sceneView];
+    [[sceneView _dynamicToolTipManager] windowChangedKeyState];
```

Comment:
Let's add a FIXME that references a bug about replacing this with something less bizarre?